file_document_import_data/file_document.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#   account_statement_file_document for OpenERP
+#   Copyright (C) 2016-TODAY Akretion <http://www.akretion.com>.
+#
+#   This program is free software: you can redistribute it and/or modify
+#   it under the terms of the GNU Affero General Public License as
+#   published by the Free Software Foundation, either version 3 of the
+#   License, or (at your option) any later version.
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#   GNU Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import fields, orm
+import base64
+
+
+class FileDocument(orm.Model):
+    _inherit = "file.document"
+
+    def get_file_document_type(self, cr, uid, context=None):
+        res = super(FileDocument, self).get_file_document_type(
+            cr, uid, context=context)
+        res.append(('import_data', 'Import Data'))
+        return res
+
+    _columns = {
+        'import_model': fields.char('Model')
```

Comment:
If you type 'purchase.order.line' in m2o your entry is saved. I also don't know functionnal name.
From db size, you save bytes if you keep an id instead of chars.

But, no strong opinion of me here but I think, one day you'll migrate to Attachment Metadata module and OCA could not accept chars for models (pending PR).